Services/CO.CDP.Organisation.WebApi.Tests/UseCase/UpdateOrganisationUseCaseTest.cs

```diff
@@ -202,6 +202,40 @@ public async Task Execute_ShouldPublishOrganisationUpdatedEvent_WhenOrganisation
 
     }
 
+    [Fact]
+    public async Task Execute_ShouldNotInsertIdentifier_WhenIdentifierIdIsEmptyOrNull()
+    {
+        var command = new UpdateOrganisation
+        {
+            Type = OrganisationUpdateType.AdditionalIdentifiers,
+            Organisation = new OrganisationInfo
+            {
+                AdditionalIdentifiers = [new OrganisationIdentifier
+                {
+                    Id = "",
+                    LegalName = "Acme",
+                    Scheme = "GB-VAT-EMPTY"
```

Comment:
This isn’t  ascheme we expect. It’s just “VAT” I think.

Reply:
Done.

Reply:
Done.